bot_api_py/events/won_round_event.py

```diff
@@ -0,0 +1,14 @@
+from bot_api_py.events.bot_event import BotEvent
+
+
+class WonRoundEvent(BotEvent):
```

Comment:
This class lacks the WonRoundEvent(int turnNumber) { super(turnNumber); } (init method) as the user should be able to read out the turn_number.

Reply:
Same as above.